### DISCUSSION TITLE
HmIP-BWTH - Urlaubsmodus wird nicht übernommen

### DISCUSSION THREAD

**Discussion by dbMerkel** [OpenCCU/OpenCCU#2711] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Beim Einstellen des Urlaubsmodus Wandthermostat HmIP-BWTH wird teileweise das Datum und die Temperatur nicht korrekt übernommen.
Zum Beispiel sind die Datenpunkte
  HmIP-RF.xxxxxxx:1.PARTY_SET_POINT_TEMPERATURE: 18.000000 und
  HmIP-RF.xxxxxxx:1.SET_POINT_TEMPERATURE: 18.000000
immer gleich. 
Setzt man die aktuelle Temperatur auf einen Wert, so wird der PARTY... auf die gleiche Temperatur gesetzt.
Eine Urlaubstemperatur ist somit nicht einstellbar.

### Describe the behavior you expected

Beim Einstellen des Urlaubsmodus Wandthermostat HmIP-BWTH wird teileweise das Datum und die Temperatur nicht korrekt übernommen.
Zum Beispiel sind die Datenpunkte
  HmIP-RF.xxxxxxx:1.PARTY_SET_POINT_TEMPERATURE: 18.000000 und
  HmIP-RF.xxxxxxx:1.SET_POINT_TEMPERATURE: 18.000000
immer gleich. 
Setzt man die aktuelle Temperatur auf einen Wert, so wird der PARTY... auf die gleiche Temperatur gesetzt.
Eine Urlaubstemperatur ist somit nicht einstellbar.

### Steps to reproduce the issue

1. Urlaubsmodus einstellen
2.Datum von bis eintragen
3.Temperatur eintragen
4. speichern (OK)

Daten Temperatur wird nicht übernommen.
Datum Teilweise auch nicht
...

### What is the version this bug report is based on?

	3.73.9.20231130

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

n/a

### Anything in the logs that might be useful for us?

```txt
Jan  7 19:28:30 ccu3-webui daemon.debug node-red[4148]: [ccu-connection:localhost]     > BidCos-RF system.multicall [""]
Jan  7 19:28:31 ccu3-webui daemon.debug node-red[4148]: [ccu-connection:localhost] rpc < http system.multicall [[{"methodName":"event","params":["nr_D24DjW_HmIP-RF","000C9D899CCB03:0","CONFIG_PENDING",false]},{"methodName":"event","params":["nr_D24DjW_HmIP-RF","000C9D899CCB03:0","RSSI_DEVICE",-60]},{"methodName":"event","params":["nr_D24DjW_HmIP-RF","000C9D899CCB03:0","DUTY_CYCLE"
Jan  7 19:28:31 ccu3-webui daemon.debug node-red[4148]: [ccu-connection:localhost]     > HmIP-RF system.multicall ["","","","",""]
Jan  7 19:28:31 ccu3-webui daemon.debug node-red[4148]: [ccu-connection:localhost] rpc < http system.multicall [[{"methodName":"event","params":["nr_D24DjW_HmIP-RF","000C9D899CCB03:1","SET_POINT_TEMPERATURE",17]},{"methodName":"event","params":["nr_D24DjW_HmIP-RF","000C9D899CCB03:1","BOOST_TIME",0]},{"methodName":"event","params":["nr_D24DjW_HmIP-RF","000C9D899CCB03:1","PARTY_MODE
Jan  7 19:28:31 ccu3-webui daemon.debug node-red[4148]: [ccu-connection:localhost]     > HmIP-RF system.multicall ["","","","","","","","","","","","","","",""]
Jan  7 19:28:31 ccu3-webui daemon.debug node-red[4148]: [ccu-connection:localhost] rpc < http system.multicall [[{"methodName":"event","params":["nr_D24DjW_HmIP-RF","000C9D899CCB03:8","EMERGENCY_OPERATION",false]},{"methodName":"event","params":["nr_D24DjW_HmIP-RF","000C9D899CCB03:8","FROST_PROTECTION",false]},{"methodName":"event","params":["nr_D24DjW_HmIP-RF","000C9D899CCB03:8",
Jan  7 19:28:31 ccu3-webui daemon.debug node-red[4148]: [ccu-connection:localhost]     > HmIP-RF system.multicall ["","","",""]
Jan  7 19:28:32 ccu3-webui daemon.debug node-red[4148]: [ccu-connection:localhost] getRegaVariables
```

**Comment by Baxxy13** [comment]
Habe kein BWTH, für ein WTH-2 kann ich das erstmal nicht reproduzieren.
![GIF 07 01 2024 20-30-22](https://github.com/jens-maus/RaspberryMatic/assets/54898249/c0909496-f204-45b4-b2dd-1db1b1433b6d)

**Comment by dbMerkel** [comment]
https://github.com/jens-maus/RaspberryMatic/assets/102617427/630f2e74-f47f-45db-b908-fb5543f46a3c

Hallo,
kann man jetzt das Problem erkennen?
Für mich sieht es so aus, als ob aus der Maske immer die Solltemperatur in den Datenpunkt Urlaubstemperatur geschrieben wird.

Ich habe auch noch HM-CC-RT-DN Geräte. Dort funktioniert alles. Die Datenpunkte sind aber anders benannt.
![DatenpunkteThermostate](https://github.com/jens-maus/RaspberryMatic/assets/102617427/627c954a-f3ea-4453-a16e-f63c6283c7d3)

**Comment by dbMerkel** [comment]
Hallo, habe gerade festgestellt, dass es beim HmIP-eTRV-C-2 auch nicht funktioniert. Dort wird auch das Datum nicht übernommen.

https://github.com/jens-maus/RaspberryMatic/assets/102617427/ea02c4fb-fdba-4238-9e07-d60c7a488eea

--HKT23 Buero HmIP-eTRV-C-2
    HmIP-RF.002CDBE9A6845F:0.CONFIG_PENDING: false
    HmIP-RF.002CDBE9A6845F:0.DUTY_CYCLE: false
    HmIP-RF.002CDBE9A6845F:0.ERROR_CODE: 0
    HmIP-RF.002CDBE9A6845F:0.LOW_BAT: false
    HmIP-RF.002CDBE9A6845F:0.OPERATING_VOLTAGE: 2.500000
    HmIP-RF.002CDBE9A6845F:0.OPERATING_VOLTAGE_STATUS: 0
    HmIP-RF.002CDBE9A6845F:0.RSSI_DEVICE: -69
    HmIP-RF.002CDBE9A6845F:0.RSSI_PEER: -62
    HmIP-RF.002CDBE9A6845F:0.SABOTAGE: false
    HmIP-RF.002CDBE9A6845F:0.UNREACH: false
    HmIP-RF.002CDBE9A6845F:0.UPDATE_PENDING: false
    HmIP-RF.002CDBE9A6845F:1.ACTIVE_PROFILE: 1
    HmIP-RF.002CDBE9A6845F:1.ACTUAL_TEMPERATURE: 20.400000
    HmIP-RF.002CDBE9A6845F:1.ACTUAL_TEMPERATURE_STATUS: 0
    HmIP-RF.002CDBE9A6845F:1.BOOST_MODE: false
    HmIP-RF.002CDBE9A6845F:1.BOOST_TIME: 0
    HmIP-RF.002CDBE9A6845F:1.CONTROL_DIFFERENTIAL_TEMPERATURE: 
    HmIP-RF.002CDBE9A6845F:1.CONTROL_MODE: 
    HmIP-RF.002CDBE9A6845F:1.DURATION_UNIT: 
    HmIP-RF.002CDBE9A6845F:1.DURATION_VALUE: 
    HmIP-RF.002CDBE9A6845F:1.FROST_PROTECTION: false
    HmIP-RF.002CDBE9A6845F:1.LEVEL: 0.700000
    HmIP-RF.002CDBE9A6845F:1.LEVEL_STATUS: 0
    HmIP-RF.002CDBE9A6845F:1.PARTY_MODE: true
    HmIP-RF.002CDBE9A6845F:1.PARTY_SET_POINT_TEMPERATURE: 0.000000
    HmIP-RF.002CDBE9A6845F:1.PARTY_TIME_END: 
    HmIP-RF.002CDBE9A6845F:1.PARTY_TIME_START: 
    HmIP-RF.002CDBE9A6845F:1.QUICK_VETO_TIME: 0
    HmIP-RF.002CDBE9A6845F:1.SET_POINT_MODE: 0
    HmIP-RF.002CDBE9A6845F:1.SET_POINT_TEMPERATURE: 20.000000
    HmIP-RF.002CDBE9A6845F:1.SWITCH_POINT_OCCURED: false
    HmIP-RF.002CDBE9A6845F:1.VALVE_ADAPTION: false
    HmIP-RF.002CDBE9A6845F:1.VALVE_STATE: 4
    HmIP-RF.002CDBE9A6845F:1.WINDOW_STATE: 0

**Comment by dbMerkel** [comment, marked as answer]
Hallo,
habe den Fehler gefunden.
Ich habe das Datum und die Uhrzeit ausgewählt und im Datum-/Uhrzeitdialog nicht den OK Button gedrückt.
Somit wurde das Datum und Uhrzeit nicht in die Datenbank geschrieben, jedoch im Urlaubsdialog angezeigt. In diesem Dialog gibt es nun nochmals einen OK-Button den ich betätigt habe, der aber nichts bewirkt.
Man muss also immer bei Startzeit 2x OK drücken und bei Endzeit 2x OK drücken. Dann funktioniert es.
Hier könnte man es ein bisschen benutzerfreundlicher machen.
1x am Ende OK.
![Urlaubsdialog](https://github.com/jens-maus/RaspberryMatic/assets/102617427/0f14c11c-e668-4b38-b53a-d53bc08510b7)
